src/DynamoPackages/PackageManagerExtension.cs

```diff
@@ -44,6 +44,14 @@ public class PackageManagerExtension : IExtension, ILogSource, IExtensionSource
         /// </summary>
         private Dictionary<string, List<PackageInfo>> NodePackageDictionary;
 
+        // TODO : Update all packages to use this hostname and replace the hostname in package manager as well.
+        /// <summary>
+        /// This will be used to match Civil 3D specific packages,
+        /// as there is a mismatch between the host dependency name in packages and the host name used by Civil 3D for Dynamo.
+        /// </summary>
+        private readonly string Civil3DHostName = "Dynamo Civil 3D";
+        private readonly string oldCivil3DHostName = "Civil 3D";
```

Comment:
hmm it's a bit weird to define this in the Dynamo code base, should we switch to a more tolerant match based on your earlier comment?

Reply:
just curious - could we do this transformation at runtime on the package manager?

Reply:
@mjkkirschner Do you mean adding the hostname in the hosts returned from Package Manager?
here: https://git.autodesk.com/Dynamo/PackageManager/blob/c1e6047b5a92419049098c4a478bfb463aed6aae/src/app.js#L239

Reply:
> 

updated with more tolerant approach

Reply:
no, I mean that when package manager encounters a package with this old hostname to just return the new one instead in the response when the user downloads this package or gets its header... but perhaps this has issues with the pkg.json?